src/data/sheltersData.ts

```diff
@@ -0,0 +1,146 @@
+type shelter = {
+  name: string;
+  address: string;
+  city: string;
+  phone: string;
+};
+
+type housingFinderResource = {
+  name: string;
+  url: string;
+  desc: string;
+  address: string;
+  phone: string;
+  phone2?: string;
+  tags2Phones?: string[];
+  email: string;
+};
+
+type additionalLinksType = {
+  name: string;
+  url: string;
+}
+
+const forMen: shelter[] = [
+  {
+    name: "Union Gospel Mission Men’s Shelter",
+    address: "318 2nd Ave Extension South,",
+    city: "Seattle, WA 98104",
+    phone: "(206) 622-5177",
+  },
+  {
+    name: "Bread of Life Mission Overnight Shelter",
+    address: "97 South Main St,",
+    city: "Seattle, WA 98104",
+    phone: "(206) 682-3579",
+  },
+  {
+    name: "DESC Queen Anne Shelter",
+    address: "157 Roy St,",
+    city: "Seattle, WA 98109",
+    phone: "(206) 283-9567",
+  },
+  {
+    name: "Congregations for the Homeless Year-Round Shelter",
+    address: "2030 3rd Ave",
+    city: "Seattle, WA 98121",
+    phone: "(206) 441-3210",
+  },
+  {
+    name: "Compass Housing Alliance Blaine Center (by referral only)",
+    address: "Denny Way near Seattle Center",
+    city: "",
+    phone: "(206) 474-1660",
+  },
+];
+
+const forWomen: shelter[] = [
+  {
+    name: "Union Gospel Mission Hope Place (for women and children)",
```

Comment:
Total random thought, but now that you've done all of the work of gathering and compiling this housing and shelter data, it'd be cool to create some type of API where similar projects that need this data can make a request and get all mens shelters, or all womens, etc.

Reply:
That would be pretty cool. As like a separate thing from cvp or as part of our offerings?